ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -44,3 +44,43 @@ nginx_sites:
       balancer: "{{ deploy_env }}_formplayer/"
       proxy_next_upstream_tries: 1
       proxy_read_timeout: 900s
+    - name: /files
```

Comment:
@snopoke i added all of these to the main nginx template because there doesnt seem to be a way to conditionally add locations. this does mean that they will be deployed to all the other environments but i do not think there is anything wrong with having them in site configs. I assume they were manually added the first time but if there was other ansible work with them done better i would be happy to change this.